modules/stitching/include/opencv2/stitching/warpers.hpp

```diff
@@ -52,17 +53,86 @@ namespace cv {
 
 /** @brief Image warper factories base class.
  */
-class WarperCreator
+
+class CV_EXPORTS_W WarperCreator
 {
 public:
-    virtual ~WarperCreator() {}
+    CV_WRAP virtual ~WarperCreator() {}
     virtual Ptr<detail::RotationWarper> create(float scale) const = 0;
 };
 
+
+class CV_EXPORTS_W PyRotationWarper
+{
+    Ptr<detail::RotationWarper> rw;
+
+public:
+    CV_WRAP PyRotationWarper(String type, float scale);
```

Comment:
I believe this should be moved out from public headers.

Could you create file similar to "shadow_umat.hpp"?

Do you have any thoughts about Java?